suites/pacific/cephadm/tier-2-verify-custom-container.yaml

```diff
@@ -0,0 +1,92 @@
+#===============================================================================================
+#-----------------------------------------------------------------------------------------------
+# Conf: conf/pacific/cephadm/tier-1_5node_cephadm_bootstrap.yaml
+# - Bootstrap
+#   - Add all the monitoring stack
+#   - Apply OSD
+# - Congifure a private registry
+# - Redeploy monitoring stack with private registry
+#===============================================================================================
+tests:
+  - test:
+      name: Install ceph pre-requisites
+      desc: installation of ceph pre-requisites
+      module: install_prereq.py
+      abort-on-fail: true
+
+  - test:
+      name: Cephadm Bootstrap
+      desc: Boostrap the cluster with monitoring stack
+      module: test_bootstrap.py
+      config:
+        command: bootstrap
+        base_cmd_args:
+          verbose: true
+        args:
+          registry-json: registry.redhat.io
+          custom_image: true
+          mon-ip: node1
+          fsid: f64f341c-655d-11eb-8778-fa163e914bcc
+          skip-monitoring-stack: true
+          orphan-initial-daemons: true
+          ssl-dashboard-port: "8445"
+          apply-spec:
+            - service_type: host
+              address: true
+              labels: apply-all-labels
+              nodes:
+                - node1
+                - node2
+                - node3
+                - node4
+            - service_type: mon
+              placement:
+                nodes:
+                  - node1
+                  - node2
+                  - node3
+            - service_type: mgr
+              placement:
+                label: mgr
+            - service_type: osd
+              service_id: all-available-devices
+              placement:
+                label: osd
+              spec:
+                data_devices:
+                  all: "true"
+            - service_type: prometheus
+              placement:
+                count: 1
+                nodes:
+                  - node1
+            - service_type: grafana
```

Comment:
Since we are deploying monitoring stack with custom images, do we need this? Also will this cause an issue when trying to deploy monitoring stacks which are already up and running?

Reply:
So basically test case is to validate the redeployment of the monitoring stack with custom images.